app/javascript/react/components/CoffeeshopTile.js

```diff
@@ -1,11 +1,20 @@
 import React from 'react'
+import { Link, Switch, Route } from 'react-router-dom'
+import CoffeeshopsShowContainer from './CoffeeshopsShowContainer'
 
 const CoffeeshopTile = (props) => {
     return(
         <div className="coffeeshop-tile">
-            <h1>{ props.name }</h1>
+            <h1>
+                <Link to={`/coffeeshops/${props.id}`}>
+                    {props.name}
+                </Link>
+            </h1>
             <p>{ props.address }</p>
             <p>{ props.city }, {props.state} { props.zip }</p>
+            <Switch>
+                <Route exact path="/coffeeshops/:id" component={CoffeeshopsShowContainer} />
+            </Switch>
```

Comment:
uhh wait, so what this code is telling me is: 
- this tile is rendered on the CoffeeShopIndex page
- it will now have a link for each shop to to the coffeeShop show page
-....but it will also conditionally render the CoffeeshopShowContainer if the path is `/coffeeshops/:id`...?

lines 15-17 dont make sense to me. As of right now, there isnt a world where that code will render anything. Unless Im mistaken, we should remove those lines